content/developers.html

```diff
@@ -256,30 +278,25 @@ <h3>Further Reading</h3>
 
 <div class="panel-heading">
                 
-<h3 class="panel-title">Git Repositories</h3>
+<h3 class="panel-title">CloudStack Git Repositories</h3>
               
 </div>
               
 <div class="panel-body">
                 
-<p>The git repositories are hosted on Apache infrastructure, and can be found here:</p>
+<p>The git repositories are hosted on GitHub infrastructure, and can be found here:</p>
 
 <ul>
-<li><a href="https://gitbox.apache.org/repos/asf/cloudstack.git" target="_blank">Apache CloudStack source code</a></li>
-<li><a href="https://gitbox.apache.org/repos/asf/cloudstack-cloudmonkey.git" target="_blank">Apache CloudStack Cloudmonkey source code</a></li>
-<li><a href="https://github.com/apache/cloudstack-ec2stack" target="_blank">Apache CloudStack EC2stack Inteface</a></li>
-<li><a href="https://github.com/apache/cloudstack-gcestack" target="_blank">Apache CloudStack GCEstack Interface</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs" target="_blank">General Documentation</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-install" target="_blank">Installation Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-admin" target="_blank">Administrative Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-rn" target="_blank">Release Notes</a></li>
+<li><a href="https://github.com/apache/cloudstack" target="_blank">Apache CloudStack source code</a></li>
+<li><a href="https://github.com/apache/cloudstack-cloudmonkey" target="_blank">Apache CloudStack Cloudmonkey source code</a></li>
+<li><a href="https://github.com/apache/cloudstack-documentation" target="_blank">Documentation</a></li>
 <li><a href="https://github.com/apache/cloudstack-www" target="_blank">Apache CloudStack Website</a></li>
 </ul>
 
 <p>To get the most recent source for Apache CloudStack, use:</p>
 
 <pre>
-git clone https://gitbox.apache.org/repos/asf/cloudstack.git
+git clone https://github.com/apache/cloudstack.git
```

Comment:
please do not merge this

Reply:
I might have over reacted in this instance. In this above piece I'd put both paths but keep in mind that apache hosts the code for legal reasons. Suggestion:
<pre>
git clone https://gitbox.apache.org/repos/asf/cloudstack.git
</pre>
<p>or the mirror at</p>
<pre>
git clone https://github.com/apache/cloudstack.git
</pre>